src/helpers/user/user-helper.js

```diff
@@ -55,9 +56,10 @@ function handleError(error, reject) {
 }
 
 export async function addUsers(usersData = { emails: [], isAdmin: undefined }) {
+  const token = await insights.chrome.auth.getToken();
```

Comment:
FWIW we should not be using the `insights` global. This was originally how the logic was migrated over from RHEcosystem in https://github.com/RedHatInsights/insights-rbac-ui/pull/1635. We can update the reducers/actions to accept a token from the useAuth hook in the future.